newIDE/app/src/ObjectsList/index.js

```diff
@@ -168,6 +171,21 @@ export default class ObjectsList extends React.Component<Props, State> {
     );
     object.setTags(getStringFromTags(this.props.selectedObjectTags));
 
+    const additionalService = ObjectsAdditionalService.getServices(
+      object,
+      objectsContainer,
```

Comment:
I'm a bit confused here 😅, objectsContainer is defined as a `gdObjectsContainer`, but in `SceneEditor`, the prop passed for it is a `gdLayout` https://github.com/4ian/GDevelop/blob/2ca593ba2b980b523aa963a5acce7f6dd0153a7c/newIDE/app/src/SceneEditor/index.js#L104 https://github.com/4ian/GDevelop/blob/2ca593ba2b980b523aa963a5acce7f6dd0153a7c/newIDE/app/src/SceneEditor/index.js#L992 Am I missing something here? Although, everything seems to work fine, but flow is complaining right now and I feel something is wrong here. :/

Reply:
`gdProject` and `gdLayout` both extends `gdObjectsContainer`, i.e: they are objects container. That should work fine with Flow as long as it's in the proper direction (i.e: you can assign a project or layout to an objects container, but not the reverse)